test_runner/batch_others/test_large_schema.py

```diff
@@ -0,0 +1,63 @@
+import time
+
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+# Test restarting page server, while safekeeper and compute node keep
+# running.
+def test_large_schema(neon_env_builder: NeonEnvBuilder):
```

Comment:
This test passes even without the code changes. I think we need a check at the end that there are no files significantly larger than expected in the data directory.

Reply:
Fixed